binaries/dump_operator_names.cc

```diff
@@ -0,0 +1,69 @@
+#include <torch/csrc/jit/script/module.h>
+#include <torch/csrc/jit/mobile/import.h>
+#include <torch/csrc/jit/mobile/module.h>
+#include <torch/csrc/jit/import.h>
+#include <torch/csrc/jit/export.h>
+#include <torch/csrc/jit/instruction.h>
+#include <torch/script.h>
+#include <c10/util/Flags.h>
+
+#include <fstream>
+
+namespace torch {
+namespace jit {
+void dump_opnames(const script::Module& m, std::unordered_set<std::string>& opnames) {
+  auto methods = m.get_methods();
+  for (const auto& method : methods) {
+    const auto &func = method.function();
+    std::cout << "function name: " << func.name() << std::endl;
+    torch::jit::Code code(func.graph());
+    for (size_t i = 0; i < code.instructions().size(); ++i) {
+      auto ins = code.instructions()[i];
+      auto node = code.instructions_source()[i];
+      if (ins.op == OpCode::OP) {
+        auto opname = node->schema().operator_name();
+        std::string namestr = opname.name;
+        if (!opname.overload_name.empty())
+          namestr += "." + opname.overload_name;
```

Comment:
is overload_name being used? the codegen script will take the output of this binary as a whitelist of root ops that need to be kept. If it needs check both opname and overload_name then probably it's easier to write them as separate yaml fields? unless opname and overload_name are guaranteed to be concatenated with "." in the codegen script as well.
we could do this in separate PR if needed.

Reply:
I think that opname and overload_name are planed to be concatenated with "." in the codegen script. Most of aten ops are already like that, except some jit prim/special ops that miss the overload_name field, but they are going to be updated to "opname.overload_name" format. Is it possible to separate the fields when parsing the yaml file?

Reply:
@smessmer Could you confirm the name.overload_name format?